src/pages/fr/core-concepts/sharing-state.md

```diff
@@ -0,0 +1,698 @@
+
+Pourtant, il existe un certain nombre d'alternatives que vous pouvez explorer. Ceux-ci inclus:
+- [Svelte's built-in stores](https://svelte.dev/tutorial/writable-stores)
+- [Solid signals](https://www.solidjs.com/docs/latest) en dehors d'un contexte de composant
+- [Envoi d'événements de navigateur personnalisés](https://developer.mozilla.org/fr/docs/Web/Events/Creating_and_triggering_events) entre les composants
+
+:::note[FAQ]
+
+<details>
+<summary>**🙋 Puis-je utiliser Nano Stores dans des fichiers `.astro` ou d'autres composants côté serveur ?
+**</summary>
+
+Nano Stores _peut_ être importé, écrit et lu dans les composants côté serveur, **mais nous ne le recommandons pas !** Cela est dû à quelques restrictions:
+- L'écriture dans un store à partir d'un fichier `.astro` ou [composant non hydraté](/fr/core-concepts/framework-components/#hydrating-interactive-components) n'affectera _pas_ la valeur reçue par [composants côté client] (/fr/reference/directives-reference/#client-directives).
+- Vous ne pouvez pas transmettre un Nano Store en tant que `prop` aux composants côté client.
+- Vous ne pouvez pas transmettre un Nano Store en tant que `prop` aux composants côté client.
+
+Si vous comprenez ces restrictions et que vous trouvez toujours un cas d'utilisation, vous pouvez essayer Nano Stores ! N'oubliez pas que les Nano Stores sont conçus pour être réactifs aux changements sur le **client** spécifiquement.
+
+</details>
+
+<details>
+<summary>**🙋 Comment Svelte Store est-il différent de Nano Stores?**</summary>
+
+**Nano Stores et [Svelte stores](https://svelte.dev/tutorial/writable-stores) sont très similaires!** En effet, [Nano Stores vous permettent d'utiliser le même raccourci `$`](https://github.com/nanostores/nanostores#svelte) pour les abonnements que vous pourriez utiliser avec Svelte stores.
+
+Si vous voulez éviter les bibliothèques tierces, [Svelte stores](https://svelte.dev/tutorial/writable-stores) sont à eux seuls un excellent outil de communication inter-îles. Néanmoins, vous préférerez peut-être Nano Stores si a) vous aimez leurs modules complémentaires pour ["objects"](https://github.com/nanostores/nanostores#maps) et [async state](https://github.com/nanostores/nanostores#lazy-stores), ou b) vous souhaitez communiquer entre Svelte et d'autres frameworks d'interface utilisateur comme Preact ou Vue.
+</details>
+
+<details>
+<summary>**🙋 Comment Solid signals est-il différent de Nano Stores?**</summary>
+
+Si vous utilisez Solid depuis un certain temps, vous avez peut-être essayé de déplacer [signals](https://www.solidjs.com/docs/latest#createsignal) ou [stores](https://www.solidjs.com/docs/latest#createstore) en dehors de vos composants. C'est un excellent moyen de partager l'état entre les Solid Islands ! Essayez d'exporter des signaux à partir d'un fichier partagé:
```

Comment:
Missing article here
```suggestion
Si vous utilisez Solid depuis un certain temps, vous avez peut-être essayé de déplacer les [signals](https://www.solidjs.com/docs/latest#createsignal) ou [stores](https://www.solidjs.com/docs/latest#createstore) en dehors de vos composants. C'est un excellent moyen de partager l'état entre les Solid Islands ! Essayez d'exporter des signaux à partir d'un fichier partagé:
```

Reply:
"entre les Îles Solid" seems a very strange phrase. What this mean?

Reply:
@thomasbnt Maybe we need something more like “Islands of Solid Components” for clarity then? “Îles de composants Solid”?

Reply:
@delucis Agreed that makes more sense

Reply:
> @thomasbnt Maybe we need something more like “Islands of Solid Components” for clarity then? “Îles de composants Solid”?

@delucis “Îles de composants Solid” may better for explanation!